internal/cli/iam/projects/invitations/invite.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package invitations
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const createTemplate = "User '{{.Username}}' invited.\n"
+
+type InviteOpts struct {
+	cli.OutputOpts
+	cli.GlobalOpts
+	username string
+	roles    []string
+	teamIds  []string
+	store    store.ProjectInviter
+}
+
+func (opts *InviteOpts) init() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+func (opts *InviteOpts) Run() error {
+	r, err := opts.store.InviteUserToProject(opts.ConfigProjectID(), opts.newInvitation())
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *InviteOpts) newInvitation() *atlas.Invitation {
+	return &atlas.Invitation{
+		Username: opts.username,
+		Roles:    opts.roles,
+		TeamIDs:  opts.teamIds,
+	}
+}
+
+// mongocli iam project(s) invitation(s) invite|create <email> --role role [--teamId teamId] [--orgId orgId].
+func InviteBuilder() *cobra.Command {
```

Comment:
nitpick: I know this pattern is everywhere and I'm not blocking but the gopher way here would have been something like `invitations.BuildCommand()` or even`invitations.NewCommand()`.

Given this package is concise we don't want to use the name of the package in constructing the only type it exposes. 

ref: https://golang.org/doc/effective_go#package-names

Reply:
After a second look I understand the word `Invite` here does not mean the package per see but one of many operations (Invite, Delete, etc) which makes me feel better, I'm still struggling with the word `Builder` since it returns a Command I _personally_ would have called it `NewInviteCommand()`

Reply:
Again I understand this is a established pattern across the code base and I would not change it